frontend/src/pages/MilestonesEditor.tsx

```diff
@@ -266,6 +270,20 @@ export const MilestonesEditor = () => {
     }
   };
 
+  const selectVersions = (e: MouseEvent | KeyboardEvent, id: number) => {
+    e.stopPropagation();
+    const containsClicked = selectedVersions.includes(id);
+    if (e.ctrlKey) {
```

Comment:
```suggestion
    if (e.ctrlKey || e.metaKey) {
```


On mac ctrl-click triggers only the context menu, so this is impossible to reach with mouse.

On mac <kbd>ctrl</kbd> is usually replaced with <kbd>command</kbd> which sets the `e.metaKey` to true. That [*might* be](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey) the <kbd>⊞ Windows</kbd> on other platforms.

I think checking also the meta key would be better than special casing the context menu.

Reply:
Thanks for pointing this out. `⊞ Windows` works with the `e.metaKey`, but I do not see it as a problem as it is rarely used in web.